todo_app/data/task.py

```diff
@@ -0,0 +1,7 @@
+#TODO - how do I make this class JSON Serializable so that I can store it in Session?
+class task():
```

Comment:
The convention in Python is for class names to be in PascalCase. i.e Task and TaskRepository

Reply:
thanks!